yarn.lock

```diff
@@ -2114,7 +2114,7 @@
     "@docusaurus/theme-search-algolia" "3.2.1"
     "@docusaurus/types" "3.2.1"
 
-"@docusaurus/react-loadable@5.5.2":
+"@docusaurus/react-loadable@5.5.2", "react-loadable@npm:@docusaurus/react-loadable@5.5.2":
```

Comment:
Seems unrelated to me

Reply:
so anyway this change has already be merged (if I rebase it should disappear)

Reply:
https://github.com/containers/podman-desktop/commit/d95a0e588f6b21620f5741712e2d2dfd733fed9c